### PR TITLE
CLI support for Vue

### PR DESCRIPTION
**Pull request by kazupon**
Issue:

## What I did
I supported `getstorybook` for vue app project with generated `vue-cli`.

## How to test
1. run `vue init [template-type] xxx `
2. run `cd xxx`
3. run `getstorybook`
4. run `npm run storybook`

## TODO (support template-type)
- [x] webpack-simple
- [x] webpack
- [x] browserify-simple
- [x] browserify
- [x] pwa

## Reference
- vue templates
https://github.com/vuejs-templates

### CODE REVIEW

**Comment by ndelangen** [comment]
Wow, you've set some goals for yourself!

Storybook is really heavy on webpack.
Is Browserify a big thing? I'm OK with sticking to supporting webpack only.

Do know the CLI needs a huge overhaul, but a generator for it like this would be fantastic to have for the Storybook for Vue launch!

Which I think can happen real soon.

File: lib/cli/lib/project_types.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ module.exports = {
   REACT_PROJECT: 'REACT_PROJECT',
   WEBPACK_REACT: 'WEBPACK_REACT',
   VUE: 'VUE',
+  WEBPACK_VUE: 'WEBPACK_VUE',
```
Comment (kazupon) [inline lib/cli/lib/project_types.js]: `WEBPACK_VUE` and `BROWSERIFY_VUE` might refactor as SFO_VUE(Single file components) due to same logic.

File: lib/cli/generators/VUE/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const mergeDirs = require('merge-dirs').default;
+const helpers = require('../../lib/helpers');
+const path = require('path');
+const latestVersion = require('latest-version');
+
+module.exports = latestVersion('@storybook/vue').then(version => {
+  mergeDirs(path.resolve(__dirname, 'template/'), '.', 'overwrite');
```
Comment (abouthiroppy) [inline lib/cli/generators/VUE/index.js]: Minor nit: I think `/` does not need in `path.resolve(__dirname, 'template/')`.

Comment (kazupon) [inline lib/cli/generators/VUE/index.js]: @abouthiroppy thank you for your reviewing!
Indeed. :)

**Comment by kazupon** [comment]
done!

**Pull request merged**